### PR TITLE
Update monster descriptions to something more interesting 

### PR DESCRIPTION
**Pull request by rtizzy**
There was a project awhile back (As in 2013 awhile back) to help improve the lore that many people contributed to including @Rivet-the-Zombie.

We ended up getting held up because extended monster descriptions wasn't in yet. 

Since that is now in, I've gone ahead and updated many of the monsters with the work everyone wrote. 

Below is the document that was being used to track this 

https://docs.google.com/document/d/15O6Kvds6EDpXzteK1RTFFOPVHCZj5oxWW906Brwb-lg/edit#

### CODE REVIEW

File: data/json/monsters.json [hunk 1]
```diff
@@ -1480,10 +1480,10 @@
     "flags": [ "DIGS", "HEARS", "POISON", "GOODHEARING", "BASHES", "DESTROYS", "WARM", "LEATHER", "FAT" ]
   },
   {
-    "id": "mon_gracke",
+    "id": "mon_gracken",
```
Comment (ZhilkinSerg) [inline data/json/monsters.json]: You should not change id of monster - I am not sure if we have a migration mechanism for monsters, also if change you change it, you should also change ids in c++ code:

```
C:\Projects\Cataclysm-DDA\src\field.cpp (1 hit)
	Line 1407:                             mtype_id( "mon_flaming_eye" ), mtype_id( "mon_kreck" ), mtype_id( "mon_gracke" ),
  C:\Projects\Cataclysm-DDA\src\mapgen.cpp (3 hits)
	Line 71: const mtype_id mon_gracke( "mon_gracke" );
	Line 71: const mtype_id mon_gracke( "mon_gracke" );
	Line 3379:                     mon_flaming_eye, mon_kreck, mon_gracke, mon_blank, mon_gozu, mon_shoggoth,
  C:\Projects\Cataclysm-DDA\src\map_extras.cpp (8 hits)
	Line 33: static const mtype_id mon_gracke( "mon_gracke" );
	Line 33: static const mtype_id mon_gracke( "mon_gracke" );
	Line 111:             mon_gelatin, mon_mi_go, mon_kreck, mon_gracke,
	Line 140:             mon_gelatin, mon_mi_go, mon_kreck, mon_gracke
	Line 176:             mon_gelatin, mon_mi_go, mon_kreck, mon_gracke
	Line 384:             mon_gelatin, mon_mi_go, mon_kreck, mon_gracke
	Line 440:             mon_gelatin, mon_flaming_eye, mon_kreck, mon_gracke, mon_blank
	Line 524:             mon_gelatin, mon_flaming_eye, mon_kreck, mon_gracke, mon_blank
```

**Comment by rtizzy** [comment]
Ah that was actually unintentional. One moment

**Comment by rtizzy** [comment]
Fixed @ZhilkinSerg, good looking out.

**Comment by John-Candlebury** [comment]
These are pretty good!

I think they would read better if you omitted the "This appears to be" part.

**Comment by nexusmrsep** [comment]
@rtizzy the convention used for texts requires two whitespaces between sentences.
See /doc/MANUAL_OF_STYLE.md, convention nr 2.

Also: 👍

**Comment by Inglonias** [comment]
I agree with John. Removing "This appears to be" makes the descriptions sound less amateurish, and shortens them. Just because we can make these descriptions much longer now doesn't mean we should.

**Comment by rtizzy** [comment]
@Inglonias 

Apologies for not responding to this sooner. I often forget to check my alerts on GH. 

This method of writing was chosen because the player character is not a "reliable narrator". 

What he is seeing may or may not be real (I.E It may be a hallucination). This is why that method of writing was chosen at the time, if that is the best possible method is debatable.

**Pull request merged**